modules/formatter-jsonapi/index.js

```diff
@@ -49,6 +49,13 @@ module.exports = function (input, opts) {
       }
     });
 
+    // Remove foreign keys from model
```

Comment:
We could cache the output of this:
https://github.com/endpoints/endpoints/blob/master/modules/formatter-jsonapi/index.js#L23

...and re-use it to do something similar.